pages/embedded-portal.tsx

```diff
@@ -49,6 +50,8 @@ const EmbeddedPortal: NextPageWithLayout<Props> = ({
     accessToken: accessToken as string,
     environmentId: environmentToken as string,
     spaceId: flatfleSpace?.id as string,
+    themeConfig: theme("#4DCA94", "#32A673"),
+    name: "Embedded",
```

Comment:
Let's try to be consistent everywhere on the naming, so `Embedded Portal` (ideally use the `workflowItems` name property from sidebar)